src/nvim/ex_docmd.c

```diff
@@ -9293,6 +9293,24 @@ static void ex_filetype(exarg_T *eap)
     EMSG2(_(e_invarg2), arg);
 }
 
+/// Same as :filetype plugin indent enable
```

Comment:
`on`, not `enable`.  `enable` is just for the `:syntax` command.  The same typo is in main.c.